### ISSUE TITLE
Window resize + Detection problem

### ISSUE THREAD

**Issue by lemoncalamitous** [SBoulanger/blazepalm#6]
I managed to make it work somehow by fixing my Tensorflow GPU issues.

However, the current solution only works for a window having 256 x 256, and changing the dimensions on codebase doesn't work. Also, the detections seem inaccurate like the snap below:

![image](https://user-images.githubusercontent.com/24726531/75674582-581d1a80-5cc0-11ea-90ba-94a72083ee4d.png)

If improvements are needed on this, I'll wait, or I'll explore. Thanks!

**Comment by SBoulanger** [comment]
Yes I believe many improvements are required for this repo (including a dynamic input size). My attention and time has been pulled to other things, so I have not been working on it. I may create a similar repo for pytorch rather than TF if one does not already exist. Although do not think I will continue with this TF implementation.

**Comment by andrerb1992** [comment]
limão calamitoso, Poderia explicar como executou o algoritmo ?

**Comment by andrerb1992** [comment]
> Consegui fazer funcionar de alguma forma corrigindo meus problemas de GPU do Tensorflow.
> 
> No entanto, a solução atual só funciona para uma janela de 256 x 256, e alterar as dimensões no codebase não funciona. Além disso, as detecções parecem imprecisas, como o snap abaixo:
> 
> ![imagem](https://user-images.githubusercontent.com/24726531/75674582-581d1a80-5cc0-11ea-90ba-94a72083ee4d.png)
> 
> Se forem necessárias melhorias nisso, vou esperar ou vou explorar. Obrigado!
